docs/source/releases/v2.1.rst

```diff
@@ -64,4 +106,8 @@ Dependency changes
 .. _deprecated_features_in_2.1:
 
 Deprecated features
-~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~
+
+- Field ``category`` removed from ``Notification`` model.
```

Comment:
Think this needs to do under a heading "Backwards incompatible changes in Oscar 2.1", after the what's new section, as it's not a deprecation but a breaking change.